src/components/WebexMediaAccess/WebexMediaAccess.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Icon,
+} from '@momentum-ui/react';
+import WebexMeetingControl from '../WebexMeetingControl/WebexMeetingControl';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Media Access component displays a prompt explaining necessary access to media.
+ *
+ * @param {object} props Data passed to the component
+ * @param {string} props.meetingID  ID of the meeting
+ * @param {string} props.media State of media controls
+ * @returns {object} JSX of the component
+ */
+export default function WebexMediaAccess({meetingID, media}) {
+  return (
+    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-media-access`}>
+      <Icon name={media === 'camera' ? 'icon-camera_36' : 'icon-microphone_36'} />
+      <h2 className="title">{media === 'camera' ? 'Allow access to camera' : 'Allow access to microphone'}</h2>
+      <p className="body">
+        Select
+        {' '}
+        <span className="allow">Allow</span>
+        {' '}
+        {media === 'camera' ? 'when your browser asks to let Webex use your camera for this video call' : 'when your browser asks to let Webex use your microphone for this video call'}
```

Comment:
This would be cleaner if you place the messages in a variable
```suggestion
        {media === 'camera' ? cameraMessage : microphoneMessage}
```